package.json

```diff
@@ -24,7 +24,7 @@
     "test:coverage": "jest --coverage --colors",
     "storybook": "start-storybook -p 9001 -s ./packages/components/src/assets -c .storybook",
     "postinstall": "preconstruct dev",
-    "example": "yarn workspace example dev",
+    "example": "yarn build && yarn workspace example dev",
```

Comment:
for nextjs to load the packages, we need to `yarn build` them first. This isn't technically correct as the examples should depend on a version of the packages and not the `main` branch of them, but I couldn't for the life of me figure out how to get yarn to pull in the packages from npm instead of symlinks. Maybe there was a caching issue, but this seems good enough for me